index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portifolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header class="section-navbar">
+      <nav>
+        <a href="#" class="toolbar-txt">Techmoves</a>
+
+          <a href="#" class="toggle-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </a>
+
+      </nav>
+    </header>
+    <section id="headline">
+
+      <div class="back-img"></div>
+      
+      <h1>Hey There.<br>I'm Phelelani</h1>
+
+      <h2>I'm a Software Developer</h2>
+
+      <p id="introduction-txt">
+        I can help you build a product, feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded,
+        don't hestiate to contact me.
+      </p>
+      
+      <div class="social-media">
+        <ul>
+          <li class="link-git"><a href="https://github.com/techmoves"><img src="images\github.png" alt="GitHub icon"></a></li>
+          <li class="link-in"><a href="https://www.linkedin.com/in/phelelani-ndlovu-883916160/"><img src="images\2.png" alt="LinkedIn icon"></a></li>
```

Comment:
- [ ] You added your LinkedIn account here, however, it seems like an angelList icon. Please add the right icon here and also don't forget to add one more list item for your angelList profile. There must be 5 icons with links according to the Figma template.